### DISCUSSION TITLE
FInd IP

### DISCUSSION THREAD

**Discussion by karthikvijay5227** [IceWhaleTech/CasaOS#1514] [General]
Is there any command in linux to find in which ip Casa OS is running as it is needed to login.My system gives different ip each time and i need to find the ip it is running for remote access.

**Comment by spuder** [comment]
It is available at http://casaos.local/

If that doesn't work its because you don't have bonjour/avahi installed. (`apt/yum install avahi`)

You can also do this from the command line

`arp -a | grep casa`

**Comment by karthikvijay5227** [comment]
I am actually Ubuntu server os in different system and I want to access the
casa os ip in another computer connected to same network.When I ran the
command arp -a | grep casa,it shown blank.Is there any casa os based
command which will show the ip on which it is running like any welcome
command?

**Comment by spuder** [comment]
Does casaos.local not work? 

```
sudo apt install -y avahi-daemon
ping casaos.local
```

You may want to consider setting up a static ip in your router so that you can predict what ip address the server will have.
